content/en/cloud_cost_management/aws.md

```diff
@@ -135,20 +135,104 @@ Attach the new S3 policy to the Datadog integration role.
 
 **Note:** Data can take up to 48 to 72 hours after setup to stabilize in Datadog.
 
-### Cost types
+## Cost types
 
-You can visualize your ingested data using the following cost types:
+Visualize your ingested data using out-of-the-box cost types. Cloud cost metric names contain various terms that describe how they are calculated.
 
-| Cost Type            | Description           |
+### Amortized vs. unblended costs
+**Amortized** cost metrics distribute some pre-payments (like up-front savings plans fees) across the entire discount term. This is also called _accrual basis_. As savings plans are applied to covered usage, the costs for that savings plan appear as usage charges. The unused portion appears as a fee on the accrual date. Not all one-time fees are amortized across the billing period. For example, the unused portion of an RI Fee always appears on the first of the month.
```

Comment:
> accrual date

Is the reader expected to know what "accrual date" means? Is it the date that a payment was applied?

Reply:
The sentence starting "Not all one-time fees..." feels like it could be a second short paragraph.

Reply:
What is an "RI Fee"?